examples/new_member.py

```diff
@@ -8,8 +8,7 @@ class MyClient(discord.Client):
         print('------')
 
     async def on_member_join(self, member):
-        guild = member.guild
-        await guild.default_channel.send('Welcome {0.mention} to {1.name}!'.format(member, guild))
+        await member.send('Welcome {0.mention} to {1.name}!'.format(member, guild))
```

Comment:
You removed the assignment of the guild variable, and then continued to use it within the format statement, this will raise a NameError exception.